shell/platform/linux/fl_engine.cc

```diff
@@ -318,7 +330,8 @@ void fl_engine_send_platform_message(FlEngine* self,
       g_object_unref(task);
       return;
     }
-  }
+  } else if (self->engine == nullptr)
+    return;
```

Comment:
Style nit: if any block of an if/else chain has braces, all must have braces, per style guide.

Reply:
I've fixed the other cases in https://github.com/flutter/engine/pull/18837.